apps/web/src/pages/studio-onboarding/components/Header.tsx

```diff
@@ -0,0 +1,87 @@
+import { css } from '@novu/novui/css';
+import { hstack, vstack } from '@novu/novui/patterns';
+import { COMPANY_LOGO_PATH } from '../../../constants/assets';
+import { Stepper } from '@mantine/core';
+import { IconCheck } from '@novu/novui/icons';
+
+export const Header = ({ active = 0 }: { active?: number }) => {
+  return (
+    <div className={css({ backgroundColor: 'surface.panel', zIndex: 1, position: 'relative', paddingBottom: '375' })}>
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please use zIndex token‏